src/main/java/picocli/CommandTokenizer.java

```diff
@@ -0,0 +1,371 @@
+package picocli;
```

Comment:
Let's put this class in a new package `picocli.interactive`. I'm thinking that this package may grow in the future to contain additional functionality for interactive command line applications.

Reply:
Except this is useful for normal as well as interactive programs. I'm planning (if you're OK with it) to add the finalized version in as an inner class of `CommandLine`, similar to everything else, and to help with the whole "one file" thing. But if I add anything else for interactive stuff, I'll do that in a separate package (and pull request).

Reply:
In normal usage the operating system shell will take care of the tokenizing and quoting so this is really for interactive use as far as I can see. There are all kinds of interesting things that can be done for picocli-based interactive shells, including things like autocompletion. I really think this should live in a new `picocli.interactive` package.

Reply:
Ok, will do. I wasn't sure if the OS handled all that or if picocli did (all my testing has been interactive, so I apparently had forgotten how non interactive apps worked 🙄).